content-src/components/DiscoveryStreamComponents/List/List.jsx

```diff
@@ -64,27 +65,23 @@ export class ListItem extends React.PureComponent {
           url={this.props.url}
         >
           <div className="ds-list-item-text">
-            <div>
+            <p>
+              <span className="ds-list-item-info clamp">
+                {this.props.domain}
+              </span>
+            </p>
+            <div className="ds-list-item-body">
               <div className="ds-list-item-title clamp">{this.props.title}</div>
               {this.props.excerpt && (
                 <div className="ds-list-item-excerpt clamp">
                   {this.props.excerpt}
                 </div>
               )}
             </div>
-            <p>
-              {this.props.context && (
-                <span>
-                  <span className="ds-list-item-context clamp">
```

Comment:
noticed we are losing out on clamp which is useful if we have longer text, should be safe to add it in DSContextFooter